services/tlsinfo/client/client.go

```diff
@@ -103,20 +106,29 @@ func (c *getCertsCmd) Execute(ctx context.Context, f *flag.FlagSet, args ...inte
 
 	foundErr := false
 	for r := range respChan {
-		fmt.Fprintf(state.Out[r.Index], "Target %s result:\n", r.Target)
 		if r.Error != nil {
 			fmt.Fprintf(state.Err[r.Index], "Get TLS certificates failure: %v\n", r.Error)
 			foundErr = true
 			continue
 		}
 		for i, cert := range r.Resp.Certificates {
-			fmt.Fprintf(state.Out[r.Index], "---Server Certificate--- %d\n", i)
-			fmt.Fprintf(state.Out[r.Index], "Issuer: %v\n", cert.Issuer)
-			fmt.Fprintf(state.Out[r.Index], "Subject: %v\n", cert.Subject)
-			fmt.Fprintf(state.Out[r.Index], "NotBefore: %s\n", time.Unix(cert.NotBefore, 0))
-			fmt.Fprintf(state.Out[r.Index], "NotAfter: %s\n", time.Unix(cert.NotAfter, 0))
-			fmt.Fprintf(state.Out[r.Index], "DNS Names: %v\n", cert.DnsNames)
-			fmt.Fprintf(state.Out[r.Index], "IP Addresses: %v\n\n", cert.IpAddresses)
+			if c.printPEM {
+				err := pem.Encode(state.Out[r.Index], &pem.Block{
```

Comment:
Lets add check is Raw has value, with error message something like "--pem flag is not supported but server version, please update it"

Reply:
Done, here's an example message:

```
$ go run ./cmd/sanssh -h -targets localhost tlsinfo get-certs -pem www.example.com:443 
0-localhost:50042: no raw cert available for CN=www.example.org,O=Internet Corporation for Assigned Names and Numbers,L=Los Angeles,ST=California,C=US, sansshell-server may be too old to return cert info
0-localhost:50042: no raw cert available for CN=DigiCert Global G2 TLS RSA SHA256 2020 CA1,O=DigiCert Inc,C=US, sansshell-server may be too old to return cert info
```